lib/jekyll-github-metadata/repository.rb

```diff
@@ -64,6 +64,10 @@ def releases_url
         "#{repository_url}/releases"
       end
 
+      def latest_release_url
+        "#{releases_url}/latest"
```

Comment:
> Or do you mean calling (if possible) latest_release.url instead of "#{releases_url}/latest" on this line?

This. Meaning can we call the `url` method of the `latest_release` object, rather than constructing the URL ourselves?